cypress/integration/testBorna/test.cy.js

```diff
@@ -0,0 +1,179 @@
+    var block_category_name = uniqid();
+    var slug_name = makeid(10);
+    beforeEach(() => cy.login());
+
+    it("Test creation of page builder block module using the New Block UI button if the is no category yet", () => {
+        cy.login();
+        cy.visit("/page-builder/page-blocks");
+        getPlaceholder("Search blocks").should("exist");
+        cy.contains("New Block").click();
+        cy.contains("Create a new block category").click();
+        cy.get(".mdc-text-field__input").eq(0).type(block_category_name);
+        cy.get(".mdc-text-field__input").eq(1).type(slug_name);
+        cy.get(".mdc-text-field__input").eq(2).type(block_category_name);
+        cy.contains("Save block category").click();
+        cy.visit("/page-builder/page-blocks");
+        getPlaceholder("Search blocks").should("exist");
+        cy.contains("New Block").click();
+        cy.contains(slug_name).click();
+        cy.contains("Save Changes").click();
+
+        cy.contains("New Block").should("exist");
+    });
+    /*
+	it("Test creation of page builder block module using the New Block UI button", () => {
+        cy.visit("/page-builder/page-blocks");
+		getPlaceholder("Search blocks").should("exist");
+		containsClick("New Block");
+		containsClick(slug_name);
+		containsClick("Save Changes");
+		cy.contains("New Block").should("exist");
+    });
+	*/
+
+    it("Test the functionality of editing the block category", () => {
+        cy.visit("/page-builder/block-categories");
+        getPlaceholder("Search block categories").should("exist");
+        cy.contains(block_category_name).click();
+        cy.get(".mdc-text-field__input")
+            .eq(2)
+            .type(block_category_name + "1");
+        cy.contains("Save block category").click();
```

Comment:
For like 90% of assertions, we use https://testing-library.com/docs/cypress-testing-library/intro/.

We try to avoid using default `cy.get` or `cy.contains` whenever possible. Use it only if CTL can't do it.

Please check the linked docs a bit. You can also check how we've approached other testing, a lot of examples there.